src/async-client/index.ts

```diff
@@ -49,29 +49,27 @@ export const asyncClient = <S extends Loadable<unknown>>(
       }
       return storeValue;
     },
+    set: (proxiedFunction, property, value) => {
+      return Reflect.set(loadable, property, value);
+    },
+    defineProperty(proxiedFunction, property, value) {
+      return Reflect.defineProperty(loadable, property, value);
+    },
     has: (proxiedFunction, property) => {
-      const proxiedFunctionHasProperty: boolean =
-        Object.prototype.hasOwnProperty.call(proxiedFunction, property);
-
-      if (proxiedFunctionHasProperty) {
-        return proxiedFunctionHasProperty;
+      if (property in proxiedFunction) {
+        return true;
```

Comment:
The `in` operator supports inherited properties whereas `hasOwnProperty` does not. We want to include inherited properties here.

(recurs)

Reply:
interesting!